controllers/users.js

```diff
@@ -28,9 +27,9 @@ router.post('/', function(req, res) {
 // Update a user
```

Comment:
So people can just add any field they want, correct? Or do they have to match the fields defined in the Model?